tests/models.ts

```diff
@@ -64,4 +64,6 @@ export type BookPageViewMode = '1up' | '2up' | 'thumb';
 
 export type ChannelSelector = 'Player' | 'Webamp';
 
-export type UserType = 'privs' | 'patron';
+export type UserType = 'privs' | 'patron' | 'guest';
```

Comment:
Suggestion: Make these user types more specific. I.e., instead of `guest`, something like "no-login".
And we need to document which privs a test account needs, and this might be very different for future tests.